Exiled.API/Extensions/MirrorExtensions.cs

```diff
@@ -292,40 +304,44 @@ private static int GetComponentIndex(NetworkIdentity identity, Type type)
         // Make custom writer(private)
         private static void MakeCustomSyncWriter(NetworkIdentity behaviorOwner, Type targetType, Action<NetworkWriter> customSyncObject, Action<NetworkWriter> customSyncVar, NetworkWriter owner, NetworkWriter observer)
         {
-            ulong dirty = 0ul;
-            ulong dirty_o = 0ul;
+            byte behaviorDirty = 0;
             NetworkBehaviour behaviour = null;
+
+            // Get NetworkBehaviors index (behaviorDirty use index)
             for (int i = 0; i < behaviorOwner.NetworkBehaviours.Length; i++)
             {
-                behaviour = behaviorOwner.NetworkBehaviours[i];
-                if (behaviour.GetType() == targetType)
+                if (behaviorOwner.NetworkBehaviours[i].GetType() == targetType)
                 {
-                    dirty |= 1UL << i;
-                    if (behaviour.syncMode == SyncMode.Observers)
-                        dirty_o |= 1UL << i;
+                    behaviour = behaviorOwner.NetworkBehaviours[i];
+                    behaviorDirty = (byte)i;
                 }
```

Comment:
Can a break be added here? I assume there won't be multiple items in behaviorOwner.NetworkBehaviours with the same type, right?

Reply:
Yes, thats more efficient, changed it.